editor/src/components/canvas/ui-jsx-canvas-renderer/ui-jsx-canvas-element-renderer-utils.tsx

```diff
@@ -248,8 +255,7 @@ function renderJSXElement(
   const elementIsIntrinsic = ElementFromScopeOrImport == null && isIntrinsicElement(jsx.name)
   const elementIsBaseHTML = elementIsIntrinsic && isIntrinsicHTMLElement(jsx.name)
   const FinalElement = elementIsIntrinsic ? jsx.name.baseVariable : ElementFromScopeOrImport
-  const finalProps =
-    elementIsIntrinsic && !elementIsBaseHTML ? filterDataProps(elementProps) : elementProps
+  const finalProps = filterProps(canvasIsLive, elementIsIntrinsic, elementIsBaseHTML, elementProps)
```

Comment:
This will only disable those handlers on something defined in this file, so an imported component with handlers will still react to mouse events, e.g.:
https://utopia.pizza/p/c27666a4-thorn-summer/?branch_name=feature/disable-user-events-on-canvas

Reply:
oh, you are right. this means that my entire approach is wrong here :D